extensions/amp-analytics/amp-analytics.md

```diff
@@ -311,8 +336,8 @@ In addition to the conditions above, `visibilitySpec` also enables certain varia
   "defaultPageview": {
     "on": "visible",
     "request": "pageview",
+    "selector": "#ad1",
     "visibilitySpec": {
-      "selector": "#anim-id",
```

Comment:
Deprecated. I like all selectors to follow exactly the same rules and be documented once. I still support them in the `visibilitySpec` for backward compatibility, but I'd like all new uses to go to the trigger.